code/drasil-theory/lib/Theory/Drasil/DataDefinition.hs

```diff
@@ -71,6 +71,9 @@ instance HasUID             DataDefinition where uid = ddQD uid uid
 instance NamedIdea          DataDefinition where term = ddQD term term
 -- | Finds the idea contained in the 'QDefinition' used to make the 'DataDefinition where'.
 instance Idea               DataDefinition where getA = either getA getA . qdFromDD
+-- | Finds the 'Quantity' defined by the 'DataDefinition'
+instance DefinesQuantity    DataDefinition where
+  defLhs = ddQDGetter defLhs defLhs
```

Comment:
What do we mean by "LHS"? Do we mean explicitly the LHS of the equation, or are we using it in the "lvalue" sense to literally mean the quantity that the model defines? (See a related discussion [here](https://github.com/JacquesCarette/Drasil/issues/3584#issuecomment-1675199708))

Reply:
The latter. I recently introduced `defLhs` in #3601, submission for a better name is open  😄.

Reply:
It seems to me that this overlaps with `HasOutput`, just without the output constraints. Is there a case where something would be an instance of `DefinesQuantity` without being one of `HasOutput`? If not, is `DefinesQuantity` not already implicit in `HasOutput`? (Of course, we may want to keep the two separate in the case of constraining functions, etc.